src/DynamoCore/DynamoCore.csproj

```diff
@@ -84,6 +84,7 @@ limitations under the License.
       <HintPath>..\..\extern\Newtonsoft.json\Newtonsoft.Json.dll</HintPath>
       <HintPath>..\..\extern\Newtonsoft.JsonBin\Newtonsoft.Json.dll</HintPath>
     </Reference>
+    <Reference Include="PresentationCore" />
```

Comment:
`PresentationCore` is a strict no-no for `DynamoCore`, let's discuss this tomorrow (perhaps we need to resort to returning `string` (that represents fully qualified resource name) and have the client (i.e. `DynamoCoreWpf` loads up a `BitmapSource`).